dns/domains.py

```diff
@@ -28,6 +28,7 @@ class Domain(DomainOptional):
     "cash": { "cname": "mcjack123.github.io" },
     "craftos-pc": { "cname": "admiring-shannon-be238c.netlify.app" },
     "metis": { "cname": "squiddev-cc.github.io" },
+    "phileos" : {"cname": "ryan-te.github.io/POS/"},
```

Comment:
This should just be the domain, not the whole path.

Reply:
ok fixed